### PR TITLE
KOGITO-1274 Change react-time-ago to moment

### PR DESCRIPTION
**Pull request by AjayJagan**

### CODE REVIEW

File: packages/management-console/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "keycloak-js": "^8.0.0",
     "lodash": "^4.17.15",
     "lower-case": "1.1.4",
+    "moment": "^2.24.0",
```
Comment (cristianonicolai) [inline packages/management-console/package.json:48]: @AjayJagan there is a wrapper for React, see https://www.npmjs.com/package/react-moment
also, please remove _@n1ru4l/react-time-ago_

Comment (AjayJagan) [reply packages/management-console/package.json]: done :+1:

File: packages/management-console/package.json [hunk 2]
```diff
@@ -52,6 +53,7 @@
     "react-event-timeline": "^1.6.3",
     "react-json-view": "^1.19.1",
     "react-json-viewer": "^2.1.0",
+    "react-moment": "^0.9.7",
```
Comment (cristianonicolai) [inline packages/management-console/package.json:55]: https://www.npmjs.com/package/react-moment#installing seems that we need to install both _react-moment_ and _moment_

File: packages/management-console/package.json [hunk 3]
```diff
@@ -32,12 +32,13 @@
   },
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@n1ru4l/react-time-ago": "^1.1.0",
+    "@patternfly/patternfly": "^2.49.0",
```
Comment (cristianonicolai) [inline packages/management-console/package.json]: @AjayJagan do we need these new dependencies?

File: packages/management-console/package.json [hunk 4]
```diff
@@ -32,12 +32,13 @@
...
     "@patternfly/react-core": "^3.140.11",
     "@patternfly/react-icons": "^3.15.3",
     "@patternfly/react-styles": "^3.7.4",
     "@patternfly/react-table": "^2.27.11",
     "@patternfly/react-tokens": "^2.8.4",
+    "@types/react-scrollbar": "^0.4.10",
```
Comment (cristianonicolai) [inline packages/management-console/package.json]: same here

File: packages/management-console/package.json (hunk 2 above)
Comment (AjayJagan) [reply packages/management-console/package.json:55]: done :+1:

File: packages/management-console/package.json (hunk 4 above)
Comment (AjayJagan) [reply packages/management-console/package.json]: removed

**Review by cristianonicolai — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Pull request merged**